core/src/components/modal/modal.tsx

```diff
@@ -820,7 +880,16 @@ export class Modal implements ComponentInterface, OverlayInterface {
         {mode === 'ios' && <div class="modal-shadow"></div>}
 
         <div role="dialog" class="modal-wrapper ion-overlay-wrapper" part="content" ref={(el) => (this.wrapperEl = el)}>
-          {showHandle && <div class="modal-handle" part="handle"></div>}
+          {showHandle && (
+            <button
+              class="modal-handle"
+              // Prevents the handle from receiving keyboard focus when it does not cycle
+              tabIndex={!isHandleCycle ? -1 : 0}
```

Comment:
I think it's `tabindex` but I also am not sure if the casing makes a difference here

Reply:
It appears that both `tabindex` and `tabIndex` casing is valid. We use both examples in multiple components. Do we have a preferred casing pattern here?

Reply:
No preference, especially if we are not being consistent already. Though it would be good to pick one and stick with it for the future.